File: support/SDKSampleJava/app/src/main/java/com/gyant/sdksamplejava/DisplayFragmentActivity.java
package com.gyant.sdksamplejava;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.v7.app.AppCompatActivity;

import com.gyant.chat_sdk.GyantChat;
import com.gyant.chat_sdk.GyantFragment;

public class DisplayFragmentActivity extends AppCompatActivity {
    private GyantFragment frag;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_generic);

        GyantChat gyantChat =  GyantChat.getInstance()
                .clientId("client_id")
                .isDev(true)
                .start();

        frag = gyantChat.createFragment();

        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.frame_layout, frag,  BuildConfig.APPLICATION_ID + ".GyantFragment")
                .commit();
    }

    @Override
    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
        this.frag.onRequestPermissionsResult(requestCode, permissions, grantResults);
    }
}

File: support/SDKSampleKotlin/app/src/main/java/com/gyant/sdksamplekotlin/DisplayFragmentActivity.kt
package com.gyant.sdksamplekotlin

import android.os.Bundle
import android.support.annotation.NonNull
import android.support.v7.app.AppCompatActivity

import com.gyant.chat_sdk.GyantChat
import com.gyant.chat_sdk.GyantFragment

class DisplayFragmentActivity : AppCompatActivity() {
    private lateinit var frag: GyantFragment

    override fun onCreate(savedInstanceState: Bundle?) {

        val gyantChat = GyantChat.getInstance()
            .clientId("client_id")
            .isDev(true)
            .start()

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_generic)

        frag = gyantChat.createFragment();

        supportFragmentManager.beginTransaction()
            .add(R.id.frame_layout, frag,  BuildConfig.APPLICATION_ID + ".GyantFragment")
            .commit();
    }

    override fun onRequestPermissionsResult(requestCode: Int, @NonNull permissions: Array<String>, @NonNull grantResults: IntArray) {
        frag.onRequestPermissionsResult(requestCode, permissions, grantResults)
    }
}
